services/src/main/java/org/exoplatform/addons/gamification/entities/domain/effective/GamificationActionsHistory.java

```diff
@@ -148,28 +148,54 @@
     query = "SELECT DISTINCT a FROM GamificationActionsHistory a where a.ruleId = :challengeId order by a.createdDate desc"
 )
 @NamedQuery(
-    name = "GamificationActionsHistory.findRealizationsByDateDescending",
+    name = "GamificationActionsHistory.findRealizationsByDateDescendingAdminAccessPrivilege",
     query = "SELECT DISTINCT g FROM GamificationActionsHistory g "
         + " WHERE g.earnerType = :type"
         + " AND g.date BETWEEN :fromDate AND :toDate"
         + " ORDER BY g.createdDate DESC"
 )
 @NamedQuery(
-    name = "GamificationActionsHistory.findRealizationsByDateAscending",
+    name = "GamificationActionsHistory.findRealizationsByDateAscendingAdminAccessPrivilege",
     query = "SELECT DISTINCT g FROM GamificationActionsHistory g"
         + " WHERE g.earnerType = :type"
         + " AND g.date BETWEEN :fromDate AND :toDate"
         + " ORDER BY g.createdDate ASC"
 )
 @NamedQuery(
-    name = "GamificationActionsHistory.findRealizationsByDateAndRules",
+    name = "GamificationActionsHistory.findRealizationsByDateAndRulesAdminAccessPrivilege",
     query = "SELECT DISTINCT g FROM GamificationActionsHistory g "
         + " WHERE g.earnerType = :type"
         + " AND g.date BETWEEN :fromDate AND :toDate"
         + " AND ((g.ruleId IS NOT NULL AND g.ruleId IN (:ruleIds)) \n"
         + "      OR (g.actionTitle IS NOT NULL AND g.actionTitle IN (:ruleEventNames))) \n"
         + " ORDER BY g.createdDate DESC"
 )
+@NamedQuery(
+    name = "GamificationActionsHistory.findRealizationsByDateDescendingUserAccessPrivilege",
```

Comment:
use filtered fields names instead of functional behavior name.
```suggestion
    name = "GamificationActionsHistory.findRealizationsByEarnerAndDateDescending",
```